### PR TITLE
Hans mnli logit

### PR DESCRIPTION
**Pull request by Jeevesh8**
## Changes
* Added doc for meaning of various labels.
* Take maximum logits over the labels corresponding to non-entailment, to obtain logit for non-entailment in HANS.

### CODE REVIEW

File: simplex/utils.py [hunk 1]
```diff
@@ -64,8 +64,8 @@ def eval(loader, model, criterion, binary=False, coeffs_t=None):
             with torch.no_grad():
                 logits = output.logits.cpu()
                 x = torch.max(torch.concat((logits[:, 0].view(-1, 1), 
-                                            logits[:, 1].view(-1, 1)), 1), 1).values
-                logits = torch.concat((logits[:, 1].view(-1,1), x.view(-1,1)), 1)
+                                            logits[:, 2].view(-1, 1)), 1), 1).values
```
Comment (RachitBansal) [inline simplex/utils.py:67]: I think you are right. I seem to have put '1' here by mistake.

File: simplex/utils.py [hunk 2]
```diff
@@ -64,8 +64,8 @@ def eval(loader, model, criterion, binary=False, coeffs_t=None):
...
+                logits = torch.concat((x.view(-1,1), logits[:, 1].view(-1,1)), 1)
```
Comment (RachitBansal) [inline simplex/utils.py]: Shouldn't this be the previous order itself? Since we want to have non-entailment as '1' and 'x' here denotes the non-entailment class labels.

Comment (Jeevesh8) [reply simplex/utils.py]: @RachitBansal Yes, it was corrected in second commit. Kindly view "Files Changed" not individual commits.
